config/local.js

```diff
@@ -1,5 +1,5 @@
 export default {
-  API_BASE_URL: 'http://192.168.40.6:5000',
+  API_BASE_URL: 'http://192.168.40.82:5000',
```

Comment:
try your best not to push `local.js` if it's not needed, everyone should have their own `local.js` saved. :+1: